docs/fr/4-Collaboration/Demander-EspaceDeNom.md

```diff
@@ -1,37 +1,21 @@
 # Aperçu
 
-Par défaut, tout le monde obtient son propre espace de nom personnel,
-`prénom-nom`. Si vous souhaitez collaborer avec votre équipe, vous pouvez
-[demander un Espace de nom](./Demander-EspaceDeNom.md) à
-[partager](Aperçu.md#share-a-namespace-in-kubeflow).
+Par défaut, chaque personne a son  espace de nom personnel, `prénom-nom`. Si vous souhaitez collaborer avec votre équipe, vous pouvez [demander un Espace de nom](./Demander-EspaceDeNom.md) à [partager](Aperçu.md#partager-le-calcul-espace-de-nom-dans-kubeflow).
```

Comment:
I don't think this is relevant anymore. Not everyone will have a personal namespace.

Reply:
It's in the English doc, I'm keepting it for now...